### ISSUE TITLE
Compatibility with SQLite SEE

### ISSUE THREAD

**Issue by vichuviswan** [ericsink/SQLitePCL.raw#330]
I see some issue with using SQLite SEE with latest version of EF core and SQLitePCL. Not sure if the issue is with EF core, or this package, or if I'm missing something.

Kindly refer the [post](https://stackoverflow.com/q/60065868/10316616) for more details. I can give you more specifics if needed.

**Comment by vichuviswan** [comment]
I did verify that the encryption and decryption is working fine with SEE command line, but fails only when integrated in my application using EFCore. Additionally, the logic is working fine until EFCore 2.2.6 and starting from 3.0 upgrade it fails and I see that its from that version EF core started using SQLitePCL package. So not sure if the issue is in the EFCore itself or with any of the dependent package its using.

Any assistance is really appreciated.

**Comment by ericsink** [comment]
First thing to check is make sure you don't have any pieces leftover from SQLitePCLRaw 1.x.  There were breaking changes in 2.0.

**Comment by vichuviswan** [comment]
Hi @ericsink 
Thanks for the response. Actually, I was not using anything explicitly from the SQLitePCLRaw 1.x package in my code, but it was one of the dependency for the EF and I guess it was used by Microsoft.Data.Sqlite. 

So for this upgrade, I did completely remove the package reference from my source and removed the installed binaries from my machine. Additionally, like I mentioned in my post in stackoverflow, I added the 2.0 package and did those 4 steps. I can not see any other obvious reference to 1.x anywhere in my code.

Appreciate your continued support to help me overcome this blocker.
